<testsuite name="registration.Flipkart Registration" tests="1" errors="1" failures="0" skipped="0" time="169.955166" timestamp="2022-11-08T23:20:54.583395" hostname="Nava"><testcase classname="registration.Flipkart Registration" name=": Login data -- @1.1 " status="failed" time="169.955166"><error type="NoSuchElementException" message="Message: no such element: Unable to locate element: {&quot;method&quot;:&quot;xpath&quot;,&quot;selector&quot;:&quot;/html/body/div[2]/div/div/div/div/div[2]/div/form/div[4]/input&quot;}&#10;  (Session info: chrome=107.0.5304.88)&#10;">
<![CDATA[
Failing step: Then we type in "Nava@18898" field ... failed in 50.094s
Location: features/registration.feature:10
Traceback (most recent call last):
  File "C:\Users\navan\AppData\Local\Programs\Python\Python310\lib\site-packages\behave\model.py", line 1329, in run
    match.run(runner.context)
  File "C:\Users\navan\AppData\Local\Programs\Python\Python310\lib\site-packages\behave\matchers.py", line 98, in run
    self.func(context, *args, **kwargs)
  File "features\steps\registrationfeatures.py", line 45, in step_impl
    context.reg.enterpass(password)
  File "C:\Users\navan\PycharmProjects\Sprint2\features\pageobjects\loginpage.py", line 42, in enterpass
    self.TypeEditBox("password_XPATH",password)
  File "C:\Users\navan\PycharmProjects\Sprint2\features\pageobjects\Base.py", line 32, in TypeEditBox
    self.driver.find_element_by_xpath(configreader.ConfigReader("locators", locator)).click()
  File "C:\Users\navan\AppData\Local\Programs\Python\Python310\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 394, in find_element_by_xpath
    return self.find_element(by=By.XPATH, value=xpath)
  File "C:\Users\navan\AppData\Local\Programs\Python\Python310\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 976, in find_element
    return self.execute(Command.FIND_ELEMENT, {
  File "C:\Users\navan\AppData\Local\Programs\Python\Python310\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "C:\Users\navan\AppData\Local\Programs\Python\Python310\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"/html/body/div[2]/div/div/div/div/div[2]/div/form/div[4]/input"}
  (Session info: chrome=107.0.5304.88)


Captured stdout:]]>
</error><system-out>
<![CDATA[
@scenario.begin
  Scenario Outline: : Login data -- @1.1 
    Given we navigate to the Flipkart url ... passed in 23.869s
    And close the login window ... passed in 5.378s
    When we hover over Login Button ... passed in 5.296s
    And Click on SignUp Link ... passed in 5.093s
    Then we validate the new user text ... passed in 0.032s
    And we type in the "9786273674" field ... passed in 60.123s
    And Click on Continue button ... passed in 20.071s
    Then we type in "Nava@18898" field ... failed in 50.094s
    And we click on the login button ... skipped in 0.000s

@scenario.end
--------------------------------------------------------------------------------

Captured stdout:









]]>
</system-out></testcase></testsuite>